Source/LinkedInWebViewController.swift

```diff
@@ -49,7 +47,14 @@ class LinkedInWebViewController: UIViewController, UIWebViewDelegate {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        webView = UIWebView(frame: view.bounds)
```

Comment:
no, in viewdidload, the view bounds are not decided yet.
You need to always use autolayout regardless. Your logic will have problem with device rotation and etc.